Detectors/ITSMFT/MFT/base/src/ChipSegmentation.cxx

```diff
@@ -42,7 +42,7 @@ ChipSegmentation::ChipSegmentation(UInt_t uniqueID):
 
   SetUniqueID(uniqueID);
 
```

Comment:
Just to be sure: does MFT need this class? Don't you use SegmentationPixel from common GeometryTGeo?

Reply:
I have corrected the comments ITS ---> MFT

I prefer to keep the T2L part, if it does not hurt too much. We may eliminate it at some point, after discussion within the MFT group.

ChipSegmentation is the segmentation of a ladder (equivalent of a stave) in chips, not the segmentation of a chip in pixels, which is the one from ITS.

Reply:
@bovulpes : thanks. It does not hurt, but I was wondering if MFT does not need a real, meaningful T2L matrix. The (common) clusterization transforms the local coordinates to tracking ones. With dummy (unit) matrices it will be the same as local coordinates.